docs/source/development.rst

```diff
@@ -329,8 +329,8 @@ Atramhasis is being developed as open source software by the
 
 Since we place a lot of importance of code quality, we expect to have a good 
 amount of code coverage present and run frequent unit tests. All commits and
-pull requests will be tested with Github Workflow Actions tests. Code coverage is being
-monitored with `Coveralls`_.
+pull requests will be tested with Github Workflow Actions tests.
+We monitor code coverage on all pull requests using [GitHub Actions] and [orgoro/coverage].
```

Comment:
Are we sure we want to have this just printed like that? Not with a link or anything?

<img width="900" height="120" alt="Image" src="https://github.com/user-attachments/assets/43e64c2b-b1e7-4ef3-a0cd-626187203758" />